test/cases/DOMRect.webidl

```diff
@@ -0,0 +1,16 @@
+// Simplified from https://drafts.fxtf.org/geometry-1/#domrect
+
+[Exposed=(Window,Worker),
+ Serializable,
+ LegacyWindowAlias=SVGRect]
+interface DOMRect /* : DOMRectReadOnly */ {
+  constructor(optional unrestricted double x = 0, optional unrestricted double y = 0,
+          optional unrestricted double width = 0, optional unrestricted double height = 0);
+
+  [NewObject] static DOMRect fromRect(optional DOMRectInit other);
+
+  inherit attribute unrestricted double x;
+  inherit attribute unrestricted double y;
+  inherit attribute unrestricted double width;
+  inherit attribute unrestricted double height;
```

Comment:
Technically `inherit` should not be there for interfaces that don't inherit another interface.

```suggestion
  attribute unrestricted double x;
  attribute unrestricted double y;
  attribute unrestricted double width;
  attribute unrestricted double height;
```